lib/indexr.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+/**
+ * Recursively creates index.js files exporting all *.js files in each directory.
+ */
+const path = require('path')
+const Promise = require('bluebird')
+const del = require('del')
+const yargs = require('yargs')
+
+const fs = Promise.promisifyAll(require('fs'))
+const glob = Promise.promisify(require('glob'))
+
+const argv = yargs
+  .usage('$0 <directory> [options]')
+  .example('$0 src', 'Recursively creates index.js in src')
+  .demand(1)
+  .help('help')
+  .alias('h', 'help')
+  .argv
+
+const FLAG = '/* @indexr */'
```

Comment:
This flag is used to identify an indexr index.js.  Index files without this flag are not overridden.